### ISSUE TITLE
Addition of Cancers/epilepsy/depression consumables 

### ISSUE THREAD

**Issue by joehcollins** [UCL/TLOmodel#1253]
Hi @andrew-phillips-1 - just wanted to open an issue related to the above modules consumables just so I have somewhere to collect any questions as I go along. This issue is linked to PR #1124 where consumables and equipment for these modules is being added. I will close this issue when I have all the consumables/equipment added.

**Comment by joehcollins** [comment]
First question - in breast cancer (or other cancers where surgery is treatment) should failure to deliver treatment due to lacking consumables still lead to scheduling of palliative care? (currently, if surgery isnt delivered due to lacking consumable availability they will not move to the post treatment HSI where palliation is scheduled)

**Comment by joehcollins** [comment]
The current logic (inclusive of consumables) would be that if someone attends `HSI_BreastCancer_Investigation_Following_breast_lump_discernible` and is able to have a biopsy (consumables are available, test succesfully detects cancer) they are either scheduled for treatment or palliation depending on staging. If the biopsy cant be delivered then they will not receive either treatment or palliation.

**Comment by andrew-phillips-1** [comment]
yes that seems right to me also

**Comment by joehcollins** [comment]
Ok thanks!

**Comment by joehcollins** [comment]
@andrew-phillips-1  - another question. For the `PostTreatmentCheck` HSIs in the cancer modules there is no additional testing using the dx_test function to determine new cancer staging (there is a note to suggest that this additional testing is potentially biopsy or ultrasound). Do we want to be using the same test again? I'm just thinking about adding in consumables for that specific HSI

**Comment by andrew-phillips-1** [comment]
Hi @joehcollins I actually didn't know whether it would be standard practice to re-do biopsies from time to time to update the staging.  Currently (at least in cervical cancer but I think all cancers) people get at most just one attempted curative treatment.  So what would be the purpose (in the model at least) of additional biopsies after attempted treatment ?  I guess for one you are thinking of reaching stage 4 and an indication for palliative care ?

**Comment by joehcollins** [comment]
Yes it was that stage 4 link with palliative care i was thinking about after treatment. However i think for now i will just leave as is and you can have a look when its all done?

**Comment by andrew-phillips-1** [comment]
yes sounds good
